ds701_book/jupyter_notebooks/19-Regression-III-More-Linear.ipynb

```diff
@@ -7,39 +7,27 @@
         "---\n",
```

Comment:
<p>Can we say a little more about why picked this formula? and what it means?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tools4ds/DS701-Course-Notes/pull/162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Reworked this section to explain what is happening with the patsy syntax.</p>